packages/api-client/src/types/integration.types.d.ts

```diff
@@ -0,0 +1,94 @@
+import { Page } from '../../../../apps/cli/src/types/index.types'
+import { IntegrationType, EventType } from '../../tests/integration.spec'
```

Comment:
Move the type definitions into here, and remove their string definitions. only enum will do

Reply:
@rajdip-b  The reason I am defining here is that If I am building api-client, the tests can find the types in build/src 

![Screenshot from 2024-08-04 20-22-31](https://github.com/user-attachments/assets/a5613c8f-6fa3-4103-8a71-0467a1dc80d1)

then if I remove it I will have to define the types as string and implicitly write the eventType and integration Types.

Reply:
This doesn't look good. It would be ideal for us to have the types defined under the `integration.types.d.ts` file